Kconfig

```diff
@@ -883,34 +883,37 @@ menu "LVGL configuration"
         config LV_USE_FS_POSIX
             bool "File system on top of posix API"
         config LV_FS_POSIX_LETTER
-            int "Set an upper cased letter on which the drive will accessible (e.g. 'A' i.e. 65)"
+            string "Set an upper cased letter on which the drive will accessible (e.g. 'A' i.e. 65)"
```

Comment:
It's not a string but a character.  But AFAIK know there is not character type in Kconfig so people need to write e.g. `65` for `'A'`. 

In v9, I'm planning to change it to work with strings.

Reply:
I see.  
I've changed them back to `int`.
Seems that the default values were missing so I've added them, [otherwise the build fails](https://github.com/lvgl/lv_micropython/runs/4958046580?check_suite_focus=true).